docs/usage-as-end-user.md

```diff
@@ -61,6 +61,27 @@ apiVersion: packet.provider.extensions.gardener.cloud/v1alpha1
 kind: ControlPlaneConfig
 ```
 
+## `WorkerConfig`
+
+The Packet extension supports specifying IDs for reserved devices that should be used for the machines of a specific worker pool.
+
+An example `WorkerConfig` for the Packet extension looks as follows:
+
+```yaml
+apiVersion: packet.provider.extensions.gardener.cloud/v1alpha1
+kind: WorkerConfig
+reservationIDs:
+- my-reserved-device-1
+- my-reserved-device-2
+reservedDevicesOnly: false
+```
+
+The `.reservationIDs[]` list contains the list of IDs of the reserved devices.
+The `.reservedDevicesOnly` field indicates whether only reserved devices should be used (based on the list of reservation IDs)
+when new machines are created.
+If it's set to `false` and the list of reservation IDs is exhausted then the next available device (unreserved) will be
+used.
+
 ## Example `Shoot` manifest
 
 Please find below an example `Shoot` manifest:
```

Comment:
Can we add an example with reservations?

Reply:
done in d514ede7cb983e27237ce6301f51ec1004785630